index.js

```diff
@@ -1,15 +1,17 @@
-const axios = require("axios");
+const axios = require('axios');
 
-const url = "https://maps.googleapis.com/maps/api/geocode/json?address=Florence";
-const myPromise = axios.get(url)
-myPromise.then(response => {
+async function consoleCityCoordinates(url) {
+  try {
+    const geoCodingResponse = await axios.get(url);
     console.log(
-      `City: ${response.data.results[0].formatted_address} -`,
-      `Latitude: ${response.data.results[0].geometry.location.lat} -`,
-      `Longitude: ${response.data.results[0].geometry.location.lng}`
+      `City: ${geoCodingResponse.data.results[0].formatted_address} -`,
+      `Latitude: ${geoCodingResponse.data.results[0].geometry.location.lat} -`,
+      `Longitude: ${geoCodingResponse.data.results[0].geometry.location.lng}`,
     );
-  })
-  .catch(error => {
+  } catch (error) {
     console.log(error);
-  });
-  
\ No newline at end of file
+  }
+}
+
+const url = 'https://maps.googleapis.com/maps/api/geocode/json?address=Florence';
```

Comment:
confusing that the name `url` is used both for this variable and as the function argument. Make this one more specific.